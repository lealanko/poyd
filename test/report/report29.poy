(* Build and report trees : margin *)
read ("1.fas")
build (1)
report (trees: (margin:1))
