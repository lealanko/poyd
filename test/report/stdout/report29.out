(t1 (((t3 t14) ((t30 t29) ((t49 t25) ((t34 t28) ((t27 (t40 t33)) (t18 ((t6
 t11) (t20 (t26 (t47 t38)))))))))) ((t9 (t8 (t32 (t5 t4)))) ((t41 (t2 t19))
 (t48 (((t7 t16) ((t15 (t36 t35)) (t23 (t24 (t21 (t37 t13)))))) (t22 ((t43
 (t45 t44)) (t12 (t31 (t46 (t10 (t17 (t42 t39))))))))))))));